Extension/package.json

```diff
@@ -11,7 +11,7 @@
     },
     "license": "SEE LICENSE IN LICENSE.txt",
     "engines": {
-        "vscode": "^1.67.0"
+        "vscode": "^1.90.0"
```

Comment:
This will cause our extension to not be available on Mac OS 12-14, Ubuntu 16-18, and Windows 7-8 and x86. Is Bob okay with that? It might drop 10% of users or something, we could look up the exact number.

Reply:
Is there a way to make this a "soft"/optional requirement? i.e. disable the feature for the unsupported VS Code versions?

Reply:
If that is an issue, we might be able to work around it by leaving this as-is and then dynamically checking for the presence of the `vscode.lm` API at runtime, although I'm not sure if VS Code blocks access to newer APIs if the package.json declaration doesn't match.

Reply:
You can break past the type checking by using:
```ts
const vscodelm = (vscode as any).lm;
```

Doing this removed some squiggles for me, but I did not test it to make sure it works. You would need to make sure it's not undefined before using it.

Reply:
Reverted and implemented Bob's suggestion. It's working well for me on my local machine, but will want to make sure to test against different configurations, and make sure we fail silently. The feature should never be enabled in the first place if vscode.lm isn't available.

Reply:
That's good news. Please make sure you test on an older version of VS Code that doesn't have the feature so we can ensure it doesn't make us crash.

Reply:
And if you miss having the types available, you could theoretically port the type definitions into our extension and use those in the LM code. I can provide some guidance if you'd like to do something like that.